### DISCUSSION TITLE
VMware exclude Power off VM

### DISCUSSION THREAD

**Discussion by netbja** [bb-Ricardo/netbox-sync#134] [Ideas]
Hello,

There is a solution to exclude from netbox-sync power off VM ? This may be a nice feature to implement...

**Comment by bb-Ricardo** [comment]
Hi, sorry for the long delay.

This should be an easy to add option.

But also means stopped VMs will constantly be deleted.

**Comment by bb-Ricardo** [comment]
Hi,

I added this option to current 'development' branch. Can you test it and give me some feedback? Please use issue #135 for that
